examples/example.krun

```diff
@@ -73,3 +73,6 @@ TEMP_READ_PAUSE = 1
 
 # CPU pinning (off by default)
 #ENABLE_PINNING = False
+
+# Autoload rmsr kernel module (on by defualt)
+# AUTOLOAD_RMSR = True
```

Comment:
Good point. At the moment, turning it off on Linux would just cause later things to go wrong. So we can remove the knob for the time being.